hw/kdrive/linux/tslib.c

```diff
@@ -0,0 +1,191 @@
+/*
```

Comment:
still required ? 
should be separate commit. And build-time configurable (because of tslib dependency)